js/lesson-25/part10.js

```diff
@@ -0,0 +1,26 @@
+`use strict`;
+
+function getTimeSeconds(h, m, s) {
```

Comment:
Wee need to check if user enter valid numbers for date  `h < 25 m < 59 s <59`

Reply:
Fixed